adapters/oidc/js/src/keycloak.js

```diff
@@ -527,9 +527,32 @@ function Keycloak (config) {
         }
 
         var access = kc.resourceAccess[resource || kc.clientId];
-        return !!access && access.roles.indexOf(role) >= 0;
+        if (!Array.isArray(obj))
```

Comment:
Since all cases here rely on the existence of the `access` variable, it would be best to add early return to avoid repeating logic.

```ts
if (!access) {
  return false;
}
```

Reply:
ok I will do the change